bash/bash-quiz.md

```diff
@@ -35,10 +35,10 @@ done
 
 #### Q3. To run a copy command in a subshell, which syntax would you use?
 
-- [x] ( command )
-- [ ] sh command
-- [ ] { command; }
-- [ ] (( command ))
+- [x] `**( command )**`
+- [ ] `**sh command**`
+- [ ] `**{ command; }**`
+- [ ] `**(( command ))**`
```

Comment:
```suggestion
- [x] `( command )`
- [ ] `sh command`
- [ ] `{ command; }`
- [ ] `(( command ))`
```
I guess we do not need additional asterics, what do you think @DevHabit ?

Reply:
Hi, @Ebazhanov .
Addressed comments.

Reply:
thanks for the quick reply! good